src/ape_run/_cli.py

```diff
@@ -46,32 +47,46 @@ def _get_command(self, filepath: Path) -> Union[click.Command, click.Group, None
             logger.error_from_exception(e, f"Exception while parsing script: {relative_filepath}")
             return None  # Prevents stalling scripts
 
+        def _get_module(filepath: Path):
+            path = ".".join(
+                (str(filepath).replace(os.path.sep, ".").split("scripts.")[-1].split(".")[:-1])
+            )
+            return path
```

Comment:
This should include the `scripts.` prefix and then you can remove it everywhere below.

Reply:
I was trying to do this but it seemed to break `Unable to load CLI endpoint for plugin 'ape_run'`

Reply:
That happens because of a syntax error or you typed something wrong.
You can use `-v debug` to see more stack trace.